blog.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="utf-8">
```

Comment:
You'll want to add the meta tag referenced in the first week. You'll want it on all pages.